package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "bugs": {
+    "url": "https://github.com/su-sws/decanter/issues"
+  },
+  "description": "A collection of SCSS patterns for theme development.",
+  "homepage": "https://github.com/su-sws/decanter",
+  "keywords": [
+    "css",
+    "mixins",
+    "sass",
+    "scss"
+  ],
+  "license": "GPLV2",
+  "name": "decanter",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/su-sws/decanter.git"
+  },
+  "version": "0.1.0",
+  "dependencies": {
+    "bourbon": "~5.0.0-beta.7",
+    "bourbon-neat": "^2.0.0-beta.2",
+    "font-awesome": "^4.6.3"
```

Comment:
These should be `devDependencies`, rather than `dependencies`, since there is no runtime. I believe this will fetch those projects into `node-modules`, which may end up throwing conflicts if the root project uses different versions. We should test this.